dev/RadioButtons/APITests/RadioButtonsTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using MUXControlsTestApp.Utilities;
+
+using Windows.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls;
+using Common;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
+{
+    [TestClass]
+    public class RadioButtonsTests : ApiTestBase
+    {
+        [TestMethod]
+        public void ValidateSelectionChangedEvent()
```

Comment:
Strangely, this test does not show up in the Test Explorer on my machine, even after rebuilding. If it shows up on your machine, this might be just some random issue with VS and we can just accept that.

Reply:
Generally, I am having troubles getting *newly* created APITests projects to work. I always dread having to create new ones here as I will almost every time be unable to actually execute the test project (VS indicated the USING_TAEF flag is set and this breaks the whole APITests project for me). It then takes some multiple VS closes, re-builds, runnign other, already existing AITests etc...until finally I can properly test them out.

Rest assured that I ran this test on VS once I finally managed to get it up successfully and verified that it fails without the supplied fix and and runs fine with the fix supplied.

Reply:
Yeah, I don't understand why its so hard/nebulus to get this set up properly.. You can run the tests using Taef instead though which ends up being easier than MSTest via VS a lot of the time

Reply:
@chingucoding I noticed that if the APITests do not show up initially, I can run the APITest "LocalizationTests" which seems to always show up. While it will probably fail, after running that test, the API tests for the control in question do show up.

If you haven't done that yet, you could see if you get the same result.

Reply:
That worked for me, awesome! Thanks for sharing that, that makes it definitely easier to get new API test projects running.

Reply:
weird.. @kmahone

Reply:
Is this an issue with Visual studio's test explorer not identifying the correct set of tests ? I'm curious if our build is not touching the correct files for VS to pick it up. Do you see it for all builds or just incremental builds. Luke fixed an issue with incremental builds recently.